src/sso/sso.ts

```diff
@@ -43,9 +43,22 @@ export class SSO {
     return `${this.workos.baseURL}/sso/authorize?${query}`;
   }
 
-  async getProfile({ code, clientID }: GetProfileOptions): Promise<Profile> {
+  async getProfile({
+    code,
+    clientID,
+    projectID,
+  }: GetProfileOptions): Promise<Profile> {
     const form = new URLSearchParams();
-    form.set('client_id', clientID);
+    if (clientID) {
+      form.set('client_id', clientID);
+    } else if (projectID) {
+      form.set('project_id', projectID);
+      this.workos.emitWarning(
+        `Use of project_id is deprecated; please use client_id in Dashboard2`,
+      );
+    } else {
+      this.workos.emitWarning(`You must enter a client_id or project_id`);
```

Comment:
Should this return and throw an error here? Or not emit a warning and let the API throw an error? It feels weird to throw a warning when it should be an error.

Reply:
@zlwaterfield anything else we shld do before merging?